docker/build_scripts/build_utils.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+# Helper utilities for build
+
+PYTHON_DOWNLOAD_URL=https://www.python.org/ftp/python
+OPENSSL_DOWNLOAD_URL=http://www.openssl.org/source
+GET_PIP_URL=https://bootstrap.pypa.io/get-pip.py
+
+
+function check_var {
+    if [ -z "$1" ]; then
+        echo "required variable not defined"
+        exit 1
+    fi
+}
+
+
+function pyver_ge {
+    # Echo 1 if first python version is greater or equal to second
```

Comment:
In that case why not just use "==2.{6,7}*" for the check?
On Jan 30, 2016 7:15 PM, "Nathaniel J. Smith" <notifications@github.com>
wrote:

> In docker/build_scripts/build_utils.sh
> <https://github.com/manylinux/manylinux/pull/17#discussion_r51354587>:
>
> > +
> > +PYTHON_DOWNLOAD_URL=https://www.python.org/ftp/python
> > +OPENSSL_DOWNLOAD_URL=http://www.openssl.org/source
> > +GET_PIP_URL=https://bootstrap.pypa.io/get-pip.py
> > +
> > +
> > +function check_var {
> > +    if [ -z "$1" ]; then
> > +        echo "required variable not defined"
> > +        exit 1
> > +    fi
> > +}
> > +
> > +
> > +function pyver_ge {
> > +    # Echo 1 if first python version is greater or equal to second
>
> The only thing this will ever be used for is to detect the strings "2.6"
> and "2.7", so as long as it isn't actually going to open a rift to the
> dungeon dimensions then I really don't care :-)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/manylinux/manylinux/pull/17/files#r51354587>.
>